pkg/reconciler/mtbroker/trigger/controller.go

```diff
@@ -101,3 +93,21 @@ func NewController(
 
 	return impl
 }
+
+// getBrokerTriggers makes sure the object passed in is a Broker, and gets all
+// the Triggers belonging to it. As there is no way to return failures in the
+// Informers EventHandler, errors are logged, and an empty array is returned in case
+// of failures.
+func getBrokerTriggers(logger *zap.SugaredLogger, triggerLister v1.TriggerLister, obj interface{}) []*eventingv1.Trigger {
```

Comment:
nit: I might call this `getTriggersForBroker` to make it more clear what this thing is doing